lib/smart_app_launch/visual_inspection_and_attestation/client/access_token_request_test.rb

```diff
@@ -0,0 +1,75 @@
+module SMARTAppLaunch
+  class AccessTokenRequestAttestationTest < Inferno::Test
+    title 'Complies with requirements for issuing access token requests'
+    id :access_token_request
+    description %(
+      Client applications comply with requirements for issuing requests for access tokens by:
+      - Issuing an HTTP POST to the EHR authorization server's token endpoint URL using content-type
+        `application/x-www-form-urlencoded`
+      - Omitting the `client_id` parameter for `confidential apps`
+    )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@62',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@72'
+
+    input :http_post_content_type,
+          title: 'Uses correct content-type in HTTP POST to the EHR authorization server\'s token endpoint URL',
```

Comment:
The input title and description should exactly mirror the test title and description